.github/workflows/get-gtest-codecov.yml

```diff
@@ -32,7 +34,7 @@ jobs:
       with:
         extra-packages: any::remotes
     - name: install local version of FIMS
-      run: remotes::install_local(upgrade = "always")
+      run: remotes::install_github("NOAA-FIMS/FIMS", upgrade = "always", force = TRUE)
```

Comment:
I think this always install the main branch, so the code coverage won't be accurate for branches, which I don't think is what we want?

Reply:
Thanks @k-doering-NOAA , the issue was resolved using `install_local(upgrade= "always", force = TRUE)`.

Reply:
fantastic!!!